t/PaymentsAdvanced/Response/FromSilentPOST.t

```diff
@@ -5,21 +5,40 @@ use Test::More;
 
 use Scalar::Util qw( blessed );
 use Test::Fatal;
+use WebService::PayPal::PaymentsAdvanced::Response::FromSilentPOST;
+();
```

Comment:
Should this be on the previous line?

Reply:
Hmm, not sure how that happened. Thanks!